quick-start-with-tidb.md

```diff
@@ -6,510 +6,532 @@ aliases: ['/docs/dev/quick-start-with-tidb/','/docs/dev/test-deployment-using-do
+Launch a local TiDB cluster with one command:
 
-    {{< copyable "shell-regular" >}}
+```shell
+tiup playground
+```
 
-    ```shell
-    source ${your_shell_profile}
-    ```
+TiUP will automatically download the necessary binaries. Wait for the success message:
 
-3. Start the cluster in the current session:
+```log
+TiDB Playground Cluster is started, enjoy!
+```
 
-    - To start a TiDB cluster of the latest version with 1 TiDB instance, 1 TiKV instance, 1 PD instance, and 1 TiFlash instance, run the following command:
+Congratulations! You're running a distributed SQL database.
 
-        {{< copyable "shell-regular" >}}
+To proceed with the next steps, open another terminal and keep `tiup` running.
 
-        ```shell
-        tiup playground
-        ```
+## Connect to TiDB
 
-    - To specify the TiDB version and the number of instances of each component, run a command like this:
+You can connect to TiDB using either the built-in TiUP client or any MySQL-compatible client.
 
-        {{< copyable "shell-regular" >}}
+### Using TiUP client
 
-        ```shell
-        tiup playground v8.4.0 --db 2 --pd 3 --kv 3
-        ```
+The TiUP client gives you a convenient way to interact with TiDB via the command line.
 
-        The command downloads a version cluster to the local machine and starts it, such as v8.4.0. To view the latest version, run `tiup list tidb`.
+```shell
+tiup client
+```
 
-        This command returns the access methods of the cluster:
+Select the endpoint to connect, and try your first commands:
 
-        ```log
-        CLUSTER START SUCCESSFULLY, Enjoy it ^-^
-        To connect TiDB: mysql --comments --host 127.0.0.1 --port 4001 -u root -p (no password)
-        To connect TiDB: mysql --comments --host 127.0.0.1 --port 4000 -u root -p (no password)
-        To view the dashboard: http://127.0.0.1:2379/dashboard
-        PD client endpoints: [127.0.0.1:2379 127.0.0.1:2382 127.0.0.1:2384]
-        To view Prometheus: http://127.0.0.1:9090
-        To view Grafana: http://127.0.0.1:3000
-        ```
+```sql
+CREATE DATABASE hello;
+USE hello;
 
-        > **Note:**
-        >
-        > + Since v5.2.0, TiDB supports running `tiup playground` on the machine that uses the Apple M1 chip.
-        > + For the playground operated in this way, after the test deployment is finished, TiUP will clean up the original cluster data. You will get a new cluster after re-running the command.
-        > + If you want the data to be persisted on storage, run `tiup --tag <your-tag> playground ...`. For details, refer to the [TiUP Reference](/tiup/tiup-reference.md#-t---tag) guide.
+CREATE TABLE hello_tidb (
+     id   INT,
+     name VARCHAR(50)
+  );
 
-4. Start a new session to access TiDB:
+INSERT INTO hello_tidb VALUES
+    (1, 'Hello World');
 
-    + Use the TiUP client to connect to TiDB.
+SELECT * FROM hello_tidb;
+```
 
-        {{< copyable "shell-regular" >}}
+You should see this output:
```

Comment:
```suggestion
The output is as follows:
```

> should, should be [link](https://developers.google.com/style/word-list#should)
Generally avoid.
Because should is ambiguous by definition, it can be problematic.
"If an outcome is expected: Describe the outcome in terms of what is expected. For example: "The process returns 10 items."

https://developers.google.com/style/word-list#should